### DISCUSSION TITLE
Issue with entry that has a comma

### DISCUSSION THREAD

**Discussion by dw-ob** [dm4t2/vue-currency-input#368] [Q&A]
I'm not sure if this is a bug but if you attempt to make an entry with a comma it changes to a decimal point.

e.g. start typing 1,000,000 and you get 1.0 and are unable to type further.

Is there a way to disable comma entry completely? So typing the above would result in 1000000 which then changes to 1,000,000 when you lose focus.

**Comment by dw-ob** [comment]
I've found setting precision to 0 will do what I want.

I'm interested to know if there's a way to do this while retaining the option to have decimal places though?

**Comment by dm4t2** [comment]
It seems you are trying to input a grouping (thousands) separator manually. This is not supported, since it will be automatically handled as a decimal separator. This is actually a feature, see #126.
